### PR TITLE
styling

### PR DESCRIPTION
**Pull request by theBwoyProgrammer**
# Styling
- Styled my app according to the Zeplin design

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,47 +1,145 @@
-.header {
+@import url('https://fonts.googleapis.com/css2?family=Bebas+Neue&display=swap');
```
Comment (danielufeli) [inline src/index.css]: - [x] Nice working on your styling, kindly make sure your design is exactly the same as the one provided.
### Your Design:
--s-design

### Template
--f-design

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @HatimDev-HE , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Good use of GitHub flow ✔️
- Clean Code ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.css (hunk 1 above)
Comment (theBwoyProgrammer) [reply src/index.css]: done

File: src/index.css [hunk 2]
```diff
@@ -1,47 +1,151 @@
...
+@import url('https://fonts.googleapis.com/css2?family=Oswald:wght@700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Abel&display=swap');
+
+body {
```
Comment (MahmoudBakr23) [inline src/index.css:4]: - [x] Well done so far styling the app. However, and as the previous reviewer required, please follow the given design in the following aspects
- Colors
- Spacings (margins and padding)
- Font sizes
- Font styles
- Font family
Keep up, you're almost there 💪 🥇

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hi @HatimDev-HE 😃, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

> Keep in mind that your long hours of work are appreciated. However, the code review process improves the code you wrote, not criticizes it. 😄

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to mention me @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.css (hunk 2 above)
Comment (theBwoyProgrammer) [reply src/index.css:4]: changed my font to` Montserrat` and `Roboto`

**Pull request merged**